src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -24,31 +24,55 @@ const PhotoDescription = (
   </div>);
 
 const MIN_SIZE = 350;
-const MIN_RATIO = 0.2;
-const MAX_RATIO = 1.7;
 const WARN_RATIO = 1.3;
 const LARGE_SCREEN = 1201;
 const MAX_DIMENSION = 1024;
 
-function getMoveBoundariesMet(canvasData, cropboxData) {
-  const { height: canvasHeight, width: canvasWidth, left: canvasLeft, top: canvasTop } = canvasData;
-  const { height: cropboxHeight, width: cropboxWidth, left: cropboxLeft, top: cropboxTop } = cropboxData;
-
-  // origin is at center
-  // coodinates are for canvas's container
-  const canvasBoundaries = {
-    minLeft: Math.round(-canvasWidth + cropboxLeft + cropboxWidth, 2),
-    maxLeft: Math.round(cropboxLeft, 2),
-    minTop: Math.round(-canvasHeight + cropboxTop + cropboxHeight, 2),
-    maxTop: Math.round(cropboxTop, 2),
+function makeMovedCanvasData({ canvasData, cropBoxData, moveX, moveY }) {
```

Comment:
What about `getCanvasDataAfterMove` or `getCanvasDataForMove`? `makeMovedCanvasData` is a little confusing to me.

Reply:
`getCanvasDataForMove` is better. Will update.